data/releases/release-cycle-schedule.yaml

```diff
@@ -0,0 +1,188 @@
+releases:
```

Comment:
We should somewhere maintain documentation how to generate the schedule from this doc. How about adding a comment on top of this file?

Reply:
Thank you so much, have tried to add one, thanks a lot!